module/move/gspread/src/actions/gspread_cell_set.rs

```diff
@@ -30,17 +34,24 @@ mod private
       ..ValueRange::default()
     };
 
-    let result = hub
+    match hub
     .spreadsheets()
     .values_update( value_range, spreadsheet_id, format!( "{}!{}", table_name, cell_id ).as_str() )
     .value_input_option( "USER_ENTERED" )
     .doit()
-    .await?
-    .1
-    .updated_cells
-    .unwrap();
+    .await
+    {
+      Ok( ( _, response) ) =>
+      {
+        match response.updated_cells
+        {
+          Some( number ) => Ok( format!( "You successfully update {} cell!", number ) ),
```

Comment:
Returning a String is not a good idea. You need to return i32 as it was. And already from the outside of the call to this function, when Ok() is received, generate a message.